sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -22,17 +27,217 @@ class TableCheckpointStore:
+        }
+        return self.table_client.create_entity(entity=my_new_entity)
+
+    @classmethod
+    def _create_ownership_entity(cls, ownership):
+        """
+        create a dictionary with the new ownership attributes so that it can be updated in tables
+        """
+        my_new_entity = {
+            u'PartitionKey': "{} {} {} Ownership".format(ownership["fullyqualifiednamespace"],
+        ownership["eventhubname"], ownership['consumergroup']),
+            u'RowKey': ownership['partitionid'],
+            u'ownerid' : ownership['ownerid'],
+        }
+        return my_new_entity
+
+    @classmethod
+    def _create_checkpoint_entity(cls, checkpoint):
+        """
+        create a dictionary with the new checkpoint attributes so that it can be updated in tables
+        """
+        my_new_entity = {
+            u'PartitionKey': "{} {} {} Checkpoint".format(checkpoint["fullyqualifiednamespace"],
+        checkpoint["eventhubname"], checkpoint['consumergroup']),
+            u'RowKey': checkpoint['partitionid'],
+            u'offset' : checkpoint['offset'],
+            u'sequencenumber' : checkpoint['sequencenumber'],}
+        return my_new_entity
+
+    def list_ownership(self, fully_qualified_namespace, eventhub_name, consumer_group):
+        """Retrieves a complete ownership list from the storage table.
+        :param str fully_qualified_namespace: The fully qualified namespace that the Event Hub belongs to.
+         The format is like "<namespace>.servicebus.windows.net".
+        :param str eventhub_name: The name of the specific Event Hub the partition ownerships are associated with,
+         relative to the Event Hubs namespace that contains it.
+        :param str consumer_group: The name of the consumer group the ownerships are associated with.
+        :rtype: Iterable[Dict[str, Any]], Iterable of dictionaries containing partition ownership information:
+                - `fully_qualified_namespace` (str): The fully qualified namespace that the Event Hub belongs to.
+                  The format is like "<namespace>.servicebus.windows.net".
+                - `eventhub_name` (str): The name of the specific Event Hub the checkpoint is associated with,
+                  relative to the Event Hubs namespace that contains it.
+                - `consumer_group` (str): The name of the consumer group the ownership are associated with.
+                - `partition_id` (str): The partition ID which the checkpoint is created for.
+                - `owner_id` (str): A UUID representing the current owner of this partition.
+                - `last_modified_time` (UTC datetime.datetime): The last time this ownership was claimed.
+                - `etag` (str): The Etag value for the last time this ownership was modified. Optional depending
+                  on storage implementation.
+        """
+        thePartitionKey = "{} {} {} Ownership".format(fully_qualified_namespace,
```

Comment:
Nit: `the` doesn't really add any additional context to variable names. I think python convention is to use snake case instead of camel case as well, so this could be renamed to `partition_key`.

Reply:
okay that make sense